### PR TITLE
Add tolerations to workers and cronjobs

### PR DESCRIPTION
**Pull request by fabubaker**
Adds `rayyan.qcri.org/qos=burstable:NoSchedule` as a toleration to workers and cronjobs.

### CODE REVIEW

**Comment by fabubaker** [comment]
> Is it good to have tolerations as a global section, as opposed to having them as part of their corresponding workloads (as in volume mounts)?

So just to clarify, you mean each worker has its own `tolerations` key?

**Comment by hammady** [comment]
> > Is it good to have tolerations as a global section, as opposed to having them as part of their corresponding workloads (as in volume mounts)?
> 
> So just to clarify, you mean each worker/cronjob has its own `tolerations` key?

Generally speaking yes, each worker definition is responsible for its own specs (images, secrets, volumes, tolerations, labels, ...). This does not mean we should repeat them, they can go to a common section for workers.

**Comment by fabubaker** [comment]
> > > Is it good to have tolerations as a global section, as opposed to having them as part of their corresponding workloads (as in volume mounts)?
> > 
> > 
> > So just to clarify, you mean each worker/cronjob has its own `tolerations` key?
> 
> Generally speaking yes, each worker definition is responsible for its own specs (images, secrets, volumes, tolerations, labels, ...). This does not mean we should repeat them, they can go to a common section for workers.

How about setting a base toleration for one of the workers/cronjobs and then using a YAML anchor for the rest?

File: helm/rayyan/values.yaml [hunk 1]
```diff
@@ -21,6 +21,11 @@ workers:
       requests:
         cpu: 400m
         memory: 2.7G
+    tolerations:
```
Comment (hammady) [inline helm/rayyan/values.yaml]: You can use yaml anchors to make it more DRY.

File: helm/rayyan/values.yaml [hunk 2]
```diff
@@ -280,6 +290,11 @@ cronjobs:
       requests:
         cpu: 500m
         memory: 0.6G
+    tolerations:
```
Comment (hammady) [inline helm/rayyan/values.yaml]: Use anchors.

**Pull request merged**